### PR TITLE
add social links, slack link

### PR DESCRIPTION
**Pull request by samundra**
#### Changes
- add facebook group link
- add slack channel link

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -39,4 +39,10 @@ be dealt in a case by case basis.
...
 pull request. Feel free to fork and update this document. Our github repository
-is [https://github.com/developers-nepal/php](https://github.com/developers-nepal/php).
\ No newline at end of file
+is [https://github.com/developers-nepal/php](https://github.com/developers-nepal/php).
+
+## Join us
+
+You can find us on:
+- [Facebook](https://www.facebook.com/groups/109070762572263/)
+- [Slack Channel](http://bit.ly/phpnp-slack)
```
Comment (geshan) [inline about.md]: footer :)

File: about.md [hunk 2]
```diff
@@ -39,4 +39,10 @@ be dealt in a case by case basis.
...
 If you feel the above information is insufficient then you can always send us
 pull request. Feel free to fork and update this document. Our github repository
-is [https://github.com/developers-nepal/php](https://github.com/developers-nepal/php).
\ No newline at end of file
+is [https://github.com/developers-nepal/php](https://github.com/developers-nepal/php).
+
+## Join us
+
+You can find us on:
+- [Facebook](https://www.facebook.com/groups/109070762572263/)
```
Comment (geshan) [inline about.md]: http://bit.ly/phpnp-fb :)

Comment (samundra) [inline about.md]: I think it's good if we keep the facebook link itself than the shortened one.

**Comment by samundra** [comment]
I removed the previous footer contents and added @geshan suggestion . Visit the pages and scroll to the bottom to the see the footer changes:

Old Footer: http://developers-nepal.github.io/php/
New Footer : http://samundra.github.io/php/

I will create PR when footer layout is finalized.

**Comment by geshan** [comment]
merging.

**Pull request merged**